docs/installation/server-config/security.rst

```diff
@@ -316,6 +318,213 @@ signed by a CA trusted by the default JVM trust store.
+   in headless mode.
+
+Role-based access control affects these actions on a KSQL cluster:
+
+- **Add**: Add a new KSQL cluster resource by using |c3| or the Confluent CLI.
+  When the cluster is created, you set more permissions on the new resources. 
+- **Contribute**: Connect and contribute to a KSQL application or cluster.
+- **Cleanup**: Delete internal data and internal topics of a KSQL application
+  or cluster.
+- **Manage**: Manage access to connect to a KSQL application or cluster. 
+
+The following table shows how RBAC affects KSQL users, applications, and
+clusters.
+
++--------------------+-------------------------+------------------------+-----------+-----------------+
+| Role               | Add a new KSQL cluster  | Connect and contribute | Cleanup   | Manage access   |
++====================+=========================+========================+===========+=================+
+| ClusterAdmin       | Yes                     | Yes                    | Yes       | Yes             |
++--------------------+-------------------------+------------------------+-----------+-----------------+
+| DeveloperManage    | No                      | No                     | Yes       | No              |
++--------------------+-------------------------+------------------------+-----------+-----------------+
+| DeveloperRead      | No                      | No                     | No        | No              |
++--------------------+-------------------------+------------------------+-----------+-----------------+
+| DeveloperWrite     | No                      | Yes                    | No        | No              |
++--------------------+-------------------------+------------------------+-----------+-----------------+
+| Operator           | No                      | No                     | Yes       | No              |
++--------------------+-------------------------+------------------------+-----------+-----------------+
+| ResourceOwner      | Yes                     | Yes                    | Yes       | Yes             |
++--------------------+-------------------------+------------------------+-----------+-----------------+
+| SecurityAdmin      | No                      | No                     | No        | No              |
++--------------------+-------------------------+------------------------+-----------+-----------------+
+| SystemAdmin        | Yes                     | Yes                    | Yes       | Yes             |
++--------------------+-------------------------+------------------------+-----------+-----------------+
+| UserAdmin          | No                      | No                     | No        | Yes             |
++--------------------+-------------------------+------------------------+-----------+-----------------+
+
+.. important::
+
+   Principals who have the ``ClusterAdmin`` role can add and register a new
+   KSQL cluster, but we recommend against giving cluster administrators access
```

Comment:
avoid using `we` ([per style guide](https://confluentinc.atlassian.net/wiki/spaces/DOC/pages/161743785/Style+Guide+for+Confluent+Documentation#StyleGuideforConfluentDocumentation-VoiceandTone))